### PR TITLE
tidb-in-kubernetes: add customizing logging and update organization

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? 

- Add warning that scaling in TiKV count might cause data loss
- Add a Fluent Bit vs Fluentd section for "Customize logging"
- Fix list format that causes chaos at the PingCAP website
- Improve organization such as splitting a para into clearer steps

### What is the related PR or file link(s)? 

- https://github.com/pingcap/docs/pull/1493
- https://github.com/pingcap/docs/pull/1500
- https://github.com/pingcap/docs/pull/1651

### Which version does your change affect? 

dev, v3.1, v3.0

### CODE REVIEW

File: dev/tidb-in-kubernetes/deploy/gcp-gke.md [hunk 1]
```diff
@@ -512,15 +546,15 @@ terraform destroy
...
+> 在执行 `terraform destroy` 过程中，可能会发生错误：`Error reading Container Cluster "tidb": Cluster "tidb" has status "RECONCILING" with message""`。当 GCP 升级 Kubernetes master 节点时会出现该问题。一旦问题出现，就无法删除集群，需要等待 GCP 升级结束，再次执行 `terraform destroy`。
 
 ### 删除磁盘
 
 如果你不再需要之前的数据，并且想要删除正在使用的磁盘，有以下两种方法可以完成此操作：
 
 - 手动删除：在 Google Cloud Console 中删除磁盘，或使用 `gcloud` 命令行工具执行删除操作。
 
-- 在执行 `terraform destroy` 之前将 Kubernetes 的 PV (Persistent Volume) 回收策略设置为 `Delete`，具体操作为在 `terraform destroy` 之前运行以下 `kubectl`命令：
+- 在执行 `terraform destroy` 之前将 Kubernetes 的 PV (Persistent Volume) 回收策略设置为 `Delete`：具体操作为在 `terraform destroy` 之前运行以下 `kubectl` 命令：
```
Comment (DanielZhangQD) [inline dev/tidb-in-kubernetes/deploy/gcp-gke.md]: why change `,` to `:`?

File: dev/tidb-in-kubernetes/deploy/gcp-gke.md [hunk 2]
```diff
@@ -550,8 +584,8 @@ terraform destroy
 
 管理多个 Kubernetes 集群的最佳实践有以下两点：
 
-1. 为每个 Kubernetes 集群创建一个新目录；
-2. 根据具体需求，使用 Terraform 脚本将上述模块进行组合。
+- 为每个 Kubernetes 集群创建一个新目录；
```
Comment (DanielZhangQD) [inline dev/tidb-in-kubernetes/deploy/gcp-gke.md]: Suggest to keep the original format, we have to create the dir first and then organize the modules in the dir.

Comment (lilin90) [reply dev/tidb-in-kubernetes/deploy/gcp-gke.md]: Got it! Will update.

File: dev/tidb-in-kubernetes/deploy/gcp-gke.md (hunk 1 above)
Comment (lilin90) [reply dev/tidb-in-kubernetes/deploy/gcp-gke.md]: @DanielZhangQD  To keep the format consistent. Can we add `自动删除：` to the beginning of this item?

**Comment by lilin90** [comment]
@DanielZhangQD All comments addressed. PTAL again~

**Pull request merged**